CONTRIBUTING.md

```diff
@@ -84,6 +84,53 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`.
 
+#### On a Windows machine:
+Install Node.js and npm. This page is helpful: http://blog.teamtreehouse.com/install-node-js-npm-windows <br>
```

Comment:
The link is helpful to install both `node` and `npm` as they now ship together in the same download. I agree on the sequencing, adding `First, ` along with `Once you've done that, ` where applicable. Thanks @kategarklavs